assets/sass/config/_variables-mui3.scss

```diff
@@ -0,0 +1,209 @@
+$c-pink-p-0: #fff;
+$c-pink-p-10: #fff7fd;
+$c-pink-p-50: #ffd7f6;
+$c-pink-p-100: #ffb6ef;
+$c-pink-p-200: #ee92da;
+$c-pink-p-300: #d276be;
+$c-pink-p-400: #b75ba2;
+$c-pink-p-500: #973b82;
+$c-pink-p-600: #80256d;
+$c-pink-p-700: #6f135b;
+$c-pink-p-800: #5d0149;
+$c-pink-p-900: #38002c;
+$c-pink-p-1000: #000;
+$c-green-g-0: #fff;
+$c-green-g-10: #f7ffe9;
+$c-green-g-50: #dfffad;
+$c-green-g-100: #c1ea7f;
+$c-green-g-200: #a4cd62;
+$c-green-g-300: #88b146;
+$c-green-g-400: #6d962b;
+$c-green-g-500: #4d760b;
+$c-green-g-600: #395d00;
+$c-green-g-700: #2b4600;
+$c-green-g-800: #1d2f00;
+$c-green-g-900: #111c00;
+$c-green-g-1000: #000;
+$c-blue-b-0: #fff;
+$c-blue-b-10: #f8faff;
+$c-blue-b-50: #dce8ff;
+$c-blue-b-100: #bed4ff;
+$c-blue-b-200: #9bb8f0;
+$c-blue-b-300: #7f9cd4;
+$c-blue-b-400: #6380b8;
+$c-blue-b-500: #446199;
+$c-blue-b-600: #2d4a82;
+$c-blue-b-700: #1b3870;
+$c-blue-b-800: #0a275f;
+$c-blue-b-900: #001745;
+$c-blue-b-1000: #000;
+$c-yellow-y-0: #fff;
+$c-yellow-y-10: #fffaef;
+$c-yellow-y-50: #ffe4b1;
+$c-yellow-y-100: #fece72;
+$c-yellow-y-200: #e1b155;
+$c-yellow-y-300: #c59539;
+$c-yellow-y-400: #aa7a1e;
+$c-yellow-y-500: #895a00;
+$c-yellow-y-600: #684500;
+$c-yellow-y-700: #4e3300;
+$c-yellow-y-800: #352300;
+$c-yellow-y-900: #1f1400;
+$c-yellow-y-1000: #000;
+$c-violet-v-0: #fff;
+$c-violet-v-10: #fbf9ff;
+$c-violet-v-50: #dbc5ff;
+$c-violet-v-100: #c4a0ff;
+$c-violet-v-200: #a983e6;
+$c-violet-v-300: #8e68cb;
+$c-violet-v-400: #6e48ab;
+$c-violet-v-500: #573195;
+$c-violet-v-600: #462083;
+$c-violet-v-700: #340e71;
+$c-violet-v-800: #340e71;
+$c-violet-v-900: #24005f;
+$c-violet-v-1000: #000;
+$c-interactive-focus: #000;
+$c-interactive-inverse-focus: #fff;
+$c-interactive-on-disable-container: rgba(3, 14, 7, 0.32);
+$c-interactive-disable-container: rgba(8, 42, 65, 0.08);
+$c-interactive-inverse-on-disable-container: rgba(255, 255, 255, 0.4);
+$c-interactive-inverse-disable-container: rgba(255, 255, 255, 0.12);
+$c-interactive-primary-hover: rgba(0, 34, 13, 0.24);
+$c-interactive-primary-focus: rgba(0, 34, 13, 0.48);
+$c-interactive-primary-press: rgba(0, 34, 13, 0.32);
+$c-interactive-secondary-hover: rgba(22, 27, 24, 0.16);
+$c-interactive-secondary-focus: rgba(22, 27, 24, 0.36);
+$c-interactive-secondary-press: rgba(22, 27, 24, 0.24);
+$c-interactive-tertiary-hover: rgba(22, 27, 24, 0.08);
+$c-interactive-tertiary-focus: rgba(22, 27, 24, 0.26);
+$c-interactive-tertiary-press: rgba(22, 27, 24, 0.16);
+
+// Custom color not from palette.
+$c-content-link: #108080;
+$c-content-primary-hover: #2e5f41;
+
+// Font sizes
+$fs-display-lg: 3.625rem; // 58px
+$fs-display-md: 2.875rem; // 46px
+$fs-display-sm: 2.375rem; // 38px
+$fs-headline-lg: 2rem; // 32px
+$fs-headline-md: 1.75rem; // 28px
+$fs-headline-sm: 1.5rem; // 24px
+$fs-title-lg: 1.25rem; // 20px
+$fs-title-md: 1rem; // 16px
+$fs-title-sm: 0.875rem; // 14px
+$fs-body-lg: 1rem; // 16px
+$fs-body-md: 0.875rem; // 14px
+$fs-body-sm: 0.75rem; // 12px
```

Comment:
Why not use the `px` values? This is particularly important for the frontend in the admin bar where the base font size `rem` are relative to is outside of our control and so we can't rely on these to map to any specific absolute values. See #888

Reply:
Sounds good. Thanks for the context! Updated!